File: public/twitteReach/src/app/SearchQuery.ts
export class SearchQuery {
	constructor(
					// Required
					public keyword: 			string,
					public screen_name: 		string,
					public category: 			string,
					public type?: 				string,

					// Not required
					public locations?: 			string[],
					public exclude_retweets?: 	boolean,
					public exclude_media?: 		boolean,
					public exclude_links?: 		boolean,
					public exclude_bots?:		boolean,
					public exclude_keywords?: 	string[],
					public auto_tweet_type?: 	string,
					public template_names?: 	string[],
					public auto_follow?: 		boolean,
					public search_potential_need_keywords?: boolean,
					public tweet_count?: 	 Number,
					public added?: 			boolean	
			){};


}

File: public/twitteReach/e2e/TweetLeads/src/main/java/com/qa/TweetLeads/base/SearchQuery.java
package com.qa.TweetLeads.base;

public class SearchQuery {
	
	public String 	keyword;
	public Boolean exclude_retweets;
	public Boolean exclude_links;
	public Boolean exclude_media;
	
	public SearchQuery(String keyword,Boolean exclude_retweets, Boolean exclude_links,Boolean exclude_media ) {
		this.keyword 			= keyword;
		this.exclude_links 		= exclude_links;
		this.exclude_retweets 	= exclude_retweets;
		this.exclude_media 		= exclude_media;
	}
}
